src/scripts/__tests__/layered-storage.test.js

```diff
@@ -83,9 +97,15 @@ describe('LayeredStorage module', () => {
   describe('LayeredStorage class', () => {
     test('LayeredStorage should getItem() from the first with a value', () => {
       const storage = new LayeredStorage([
-        { getItem (key) { return null; } },
-        { getItem (key) { return 'a'; } },
-        { getItem (key) { return 'b'; } }
+        {getItem (key) {
+          return null; 
+        }},
+        {getItem (key) {
+          return 'a'; 
+        }},
+        {getItem (key) {
+          return 'b'; 
+        }}
```

Comment:
This really feels like a pretty ugly result. :\

Reply:
I agree. Adding "allowSingleLine" fixed these